docs/Git_Reference.md

```diff
@@ -0,0 +1,117 @@
+[**Table of Contents**](index.md)
+
+# Git Info
+
+![git info](md_Graphics/git_sm.jpg)
+
+## Getting Started
+* [Git Basics](https://git-scm.com/book/en/v2/Getting-Started-Git-Basics)
+* [Git Book](https://git-scm.com/book/en/v2)
+* [Git Bootcamp and Cheat Sheet](https://docs.python.org/devguide/gitbootcamp.html)
+
+### General Information
+* Snapshots, not Differences.
+
+* Nearly every operation is local.
+
+* Integrity - SHA-1 Hash (40 Characters).
+
+* 3 States that files can reside in:
+    * Committed - Safely stored in the local database.
+    * Modified - Changed, not commited.
+    * Staged - Marked to go into the next commit.
+
+* fetch imports commits from a remote repo into your local repo.
+* Commits are stored as remote branches instead of normal local branches.
+* pull = fetch + merge
+* git pull --rebase  origin   ← Same as git pull but instead of git merge, git rebase
+* git pull --rebase  origin  == fetch + rebase
+* Pull Request is a mechanism for a developer to notify team members that they have completed a feature.
```

Comment:
This is more specific to github not git.

Reply:
True, I removed the line.